text/0000-rfc-process.md

```diff
@@ -0,0 +1,232 @@
+* ensures the motivation for a change is clear.
+* ensures the impact of a change on users is clear and migration path and backwards compatibility are considered.
+* aligns stakeholders on any changes to the user interface(s) and/or APIs (e.g. pack user interface, platform API, buildpack API).
+* aligns stakeholders on any substantive architectural changes.
+* aligns stakeholders on any processes or workflows adopted by the project.
+* provides visibility to the community regarding ongoing work and upcoming changes.
+* provides a mechanism by which any interested party can provide early feedback on an upcoming change.
+* provides a version controlled record of our decisions and the thought-process behind them.
+* is open to anyone! We enthusiastically welcome RFCs from authors that have no formal role in project governance (yet ;p).
+
+The RFC process **is not**:
+* a replacement for high-quality user-facing documentation(although high-quality RFCs enable the creation of high-quality documentation).
+* a transaction log of changes. Readers should be able to understand the change proposed without undue reference to previous RFCs.
+* a feature request process. RFCs require a level of design and implementation detail that goes beyond a feature request. Pure feature requests should instead be initiated as discussions on the community repo, issues on this repo, or issues on component repos. These requests may serve as the motivation for future RFCs.
+
+
+# Definitions
+[definitions]: #definitions
+
+**Technical Oversight Committee**: The governing body responsible for overseeing the project as described in our governance document.
+
+**Team**: A group responsible for a particular area of the project.
+
+**Team Lead**: A team maintainer with special responsibilities for representing the concerns of a team to the project, including casting binding votes on project RFCs.
+
+**RFC**: A document proposing a substantive change to the project as described in this document.
+
+**Project RFC**: An RFC with cross-cutting implications, requiring collaboration between multiple teams or affecting multiple personas.
+
+**Team RFC**: An RFC with narrower implication in comparison to a project RFC, with work scoped to a single team and implications for a narrower set of personas.
+
+**Author**: The author or authors of an RFC are responsible for producing the draft RFC and updating it to incorporate feedback. Changes should not be made to a draft RFC without the author's consent.
+
+**Steward**: The steward of an RFC is responsible for shepherding an RFC through the process, including working with the author to ensure RFC completeness and quality, and building consensus among stakeholders.
+
+**Call for Votes**: When an RFC is deemed ready by a team lead, that team lead initiates the voting process with a call for votes. At this point the RFC is closed to modification.
```

Comment:
Should this be the "team lead" or the "Steward"?

Reply:
I like steward... made suggestion